solutions/devsprint-gabriel-goto-1/app/src/main/java/com/devpass/spaceapp/data/model/LaunchPadDetailsResponse.kt

```diff
@@ -0,0 +1,22 @@
+package com.devpass.spaceapp.data.model
+
+import com.google.gson.annotations.SerializedName
+
+data class LaunchPadDetailsResponse(
+    @SerializedName("id")
+    val id: String,
+    @SerializedName("name")
+    val name: String,
+    @SerializedName("locality")
+    val locality: String,
+    @SerializedName("region")
+    val region: String,
+    @SerializedName("latitude")
+    val latitude: Double,
+    @SerializedName("longitude")
+    val longitude: Double,
+    @SerializedName("launch_attempts")
+    val launchAttempts: Int,
+    @SerializedName("launch_successes")
+    val launchSuccesses: Int
```

Comment:
Poderiamos usar trailling commas aqui : )
```suggestion
    val launchSuccesses: Int,
```

Reply:
Opa, esqueci :S, obrigada!